kobo/device.go

```diff
@@ -410,3 +410,20 @@ func (d Device) StorageGB() int {
 	}
 	panic("unknown device")
 }
+
+// DisplayPPI returns the display Pixels Per Inch (PPI) of a Device
+func (d Device) DisplayPPI() int {
+	switch d {
+	case DeviceTouchAB, DeviceTouchC, DeviceTouch2:
+		return 167
+	case DeviceMini:
+		return 200
+	case DeviceAura, DeviceGlo, DeviceAuraEdition2v1, DeviceAuraEdition2v2:
+		return 213
+	case DeviceAuraHD, DeviceAuraH2O, DeviceAuraH2OEdition2v1, DeviceAuraH2OEdition2v2:
+		return 265
+	case DeviceClaraHD, DeviceLibraH2O, DeviceAuraONE, DeviceForma, DeviceGloHD, DeviceAuraONELimitedEdition, DeviceForma32:
+		return 300
+	}
```

Comment:
@NiLuJe, can you confirm these values (Wikipedia's been wrong about quite a few Kobo specs before)?

Reply:
The touch devices are 600 x 800 at 6"
The Mini is 600 x 800 at 5"
Aura and Glo's are 7(6|5)8 x 1024 at 6 "
ClaraHD, Libra, Aura One, Forma are all 300PPI according to [Kobo](https://us.kobobooks.com/pages/compare-ereaders)

And the PPI of 265 for the Aura HD and H2O sounds correct from my memory. In other words, I think they're all close enough.

Reply:
Ok, the wayback machine has a few older ones https://web.archive.org/web/20161127032346/https://us.kobobooks.com/pages/compare-ereaders

Looks like the aura/glo class devices should be 212 PPI

Reply:
I don't actually remember what I based this on, but this should be correct (hopefully ;p): https://github.com/NiLuJe/FBInk/blob/04903cbc2c60f574b6b2af88878aa8eb2369110a/fbink_device_id.c#L565-L787

Reply:
TL;DR: At a quick glance, the latest iteration looks good ;).